Detections/AuditLogs/GuestAccountsAddedinAADGroupsOtherThanTheOnesSpecified.yaml

```diff
@@ -0,0 +1,53 @@
+id: 1caa690e-4d5c-4083-ba1a-97cdebad4eff
+name: Guest accounts added in AAD Groups other than the ones specified
+description: |
+  'Guest Accounts are added in the Organization Tenants to perform various tasks i.e projects execution, support etc.. This detection notifies when guest users are added to Azure AD Groups other than the ones specified and poses a risk to gain access to sensitive apps or data.'
+severity: High
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+queryFrequency: 5m
```

Comment:
queryFrequency determines how often the query runs against the data. Wondering if you wanted it to be higher than 5 minutes.

Reply:
5m is ok to be as default. Since it can be customized.